app/models/organization.rb

```diff
@@ -36,11 +35,13 @@ def self.counties
     Organization.select(:county).map(&:county).uniq
   end
 
-  private def fetch_geocoding_data
+  private
+
+  def fetch_geocoding_data
     begin
-      result = Geocoder.search(address).first
+      result = Geocoder.search(addresses.first).first
```

Comment:
How about a new `primary_address` method instead of all the `addresses.first`? I feel that is more readable and gives us the nice flexibility to be able to change what a `primary_address` means if we want to (like if we want it explicitly selected rather than simply the first in the list). Also, rather than simply grabbing the first, we may want to grab the first after sorting by id, as sort order is unpredictable if it isn't explicitly defined in an order by clause.